B/Bullet/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Bullet"
+version = v"3.22.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/bulletphysics/bullet3/archive/refs/tags/3.22b.tar.gz",
+                  "c6cd89ecbc4bd73fee64723c831c1578daab056d88774755a6f56afc6f417b2b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd bullet*
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -std=c++0x" \
```

Comment:
If you have to specify the C++ standard, it's a good indication the upstream project should have done that.  Packagers should never have to do that manually.  Also, you may want to use a more recent GCC version.  What's the standard they actually need?

Reply:
[This](https://github.com/bulletphysics/bullet3#requirements-for-bullet-physics-c) is all the documentation I've been able to find: "A C++ compiler for C++ 2003. The library is tested on Windows, Linux, Mac OSX, iOS, Android, but should likely work on any platform with C++ compiler." Should I create an issue over there?

Reply:
Yes, they should set `CXX_STANDARD`, see for example https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html or https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/.  Note that there is no C++ 2003 standard in CMake, what they need is probably 2011.

You can try requiring GCC 5 below and here doing
```suggestion
    -DCMAKE_CXX_STANDARD="11" \
```

Reply:
Look at that, you magically fixed PowerPC!